<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Replay</class>
 <widget class="QWidget" name="Main_Replay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1041</width>
     <height>541</height>
    </rect>
   </property>
   <widget class="QLabel" name="LB_curTime">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>490</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BT_PlayOrPause">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>490</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>播放</string>
    </property>
   </widget>
   <widget class="QSlider" name="videoTimeSilder">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>460</y>
      <width>781</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>490</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1X</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.5X</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2X</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="LB_totalTime">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>490</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="video_label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>781</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
